python/pyspark/errors/error_classes.py

```diff
@@ -79,11 +114,26 @@
       "returnType can not be specified when `<arg_name>` is a user-defined function, but got <return_type>."
     ]
   },
+  "CANNOT_UNPERSIST_BROADCAST": {
```

Comment:
is it possible to combine `CANNOT_UNPERSIST_BROADCAST` `CANNOT_REDUCE_BROADCAST` and `CANNOT_DESTROY_BROADCAST` to a single `INVALID_BROADCAST_OPERATION` with parameter `operation`

Reply:
Sounds good. Seems like we can consolidate `CANNOT_DESTROY_BROADCAST` as well :-)